config/README.md

```diff
@@ -104,6 +104,11 @@ There are 4 files you must provide:
 > filtering), you can do that, but make sure the formats of any modified
 > resource files meet the formatting expectations outlined above!
 
+In addition, if you want to generate
+[ASCII-style alignment pileups][ascii-pileups], a **BED6** file following the
+[Ensembl format][ensembl-bed] with the regions of interest you want the pileups
+to be performed on.
+
```

Comment:
Not sure why this is copied over from `README.md`? Is the entire section copied? Is this really necessary? Best to have only a single source of truth, otherwise we will forget updating both copies sooner or later if/when something changes (probably sooner)

Reply:
It is not 100% copied, but part of it. This README is required by the Snakemake Catalogue.  Only a few things should be modified there - if necessary - in the future. What should I do with it?